style.css

```diff
@@ -0,0 +1,918 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:wght@300;400;500;600;700;800&family=Inter:wght@100;300;400;500;600;700;800&display=swap');
+
+:root {
+  --primary-font: "Crete Round", sans-serif;
+  --secondary-font: "Inter", sans-serif;
+  --primary-color: #ff6b00;
+  --secondary-color: #f55800;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  border: 1 solid green;
+}
+
+body {
+  font-family: sans-serif;
+}
+
+.header-container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+}
+
+.navbar {
+  position: relative;
+  background: #3e3c3c;
+  padding: 2% 8%;
+  height: 72px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-links {
+  height: 30px;
+  list-style: none;
+  align-items: center;
+}
+
+.nav-links li {
+  margin-left: 2.5rem;
+  display: inline-block;
+}
+
+.nav-links li a {
+  color: #fff;
+  text-decoration: none;
+  font-family: var(--secondary-font);
+  font-size: 15px;
+  font-weight: 600;
+  transition: 0.3s;
+}
+
+.nav-links li a:hover {
+  color: #e05304;
+}
+
+.my-logo {
+  width: 80px;
+  height: 60px;
+}
+
+.fa-bars {
+  display: none;
+  color: white;
+  width: 24px;
+  height: 24px;
+  cursor: pointer;
+}
+
+.main-container {
+  height: calc(100% - 72px);
+  width: 100%;
+  position: relative;
+  padding-left: 36px;
+  background-image: url(images/Illustration-header.png);
+  background-repeat: no-repeat;
+  background-position: right;
+  display: flex;
```

Comment:
- [ ] Good work using GitHub inline features to write your comment. Good work explaining why the change is necessary. Your comment however gets confusing toward the end. Did you give a good example of DRY code / just highlighted repetition in the student's code under a good example? 🤔  I suggest you write a good example of dry code using the `display: flex` property instead of quoting the student's code since this has already been done at the beginning g of the comment